tests/sqllogictests/suites/query/02_function/02_0066_function_try_to_decimal

```diff
@@ -0,0 +1,14 @@
+statement ok
+CREATE TABLE test (a double) ENGINE=Memory
+
+statement ok
+INSERT INTO test VALUES (0.0941881516339449), (0.8833429252655464)
+
+query F 
+SELECT try_to_decimal(20, 19)(a) FROM test;
```

Comment:
Simple way:

```
select try_to_decimal(20, 19)(a) from (values(0.0941881516339449),(0.8833429252655464))t(a);
```

Reply:
select values from table is better, as literal value is always parsed as a decimal value